components/brave_new_tab_ui/components/default/settings/index.ts

```diff
@@ -180,10 +181,16 @@ export const SettingsSidebarButton = styled<SettingsSidebarButtonProps, 'button'
       color: ${p => p.theme.color.brandBrave};
     }
   }
+
+  &:focus {
```

Comment:
Please can we remove this until we have a solution? Otherwise we're leaving non-mouse users out in the dark.

Reply:
done